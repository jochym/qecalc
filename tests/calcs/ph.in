Phonon of MgB2
&INPUTPH
    tr2_ph = 1.0d-10,
    amass(1) = 24.305,
    amass(2) = 11.000,
    prefix = 'mgb2',
    outdir = 'temp/',
    fildyn = 'mgb2.dynG',
    fildrho = 'mgb2.drho_G',
/
0.0 0.0 0.0
