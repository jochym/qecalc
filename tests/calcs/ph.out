
     Program PHONON    v.4.1    starts ...
     Today is 18Dec2009 at 20: 9: 7 

     Parallel version (MPI)

     Number of processors in use:       8
     K-points division:     npool     =    8

     Ultrasoft (Vanderbilt) Pseudopotentials

     Planes per process (thick) : nr3 = 18 npp =  18 ncplane =  256
 
     Proc/  planes cols     G    planes cols    G      columns  G
     Pool       (dense grid)       (smooth grid)      (wavefct grid)
        1    18    163     1835   18    163     1835     61      395
 

     Check: negative/imaginary core charge=   -0.000180    0.000000

                                                                                

     bravais-lattice index     =            4
     lattice parameter (a_0)   =       6.0000  a.u.
     unit-cell volume          =     204.9350 (a.u.)^3
     number of atoms/cell      =            3
     number of atomic types    =            2
     kinetic-energy cut-off    =      16.0000  Ry
     charge density cut-off    =      64.0000  Ry
     convergence threshold     =      1.0E-10
     beta                      =       0.7000
     number of iterations used =            4
     Exchange-correlation      =  SLA  PW   PBE  PBE (1434)


     celldm(1)=    6.00000  celldm(2)=    1.00000  celldm(3)=    1.09555
     celldm(4)=   -0.50000  celldm(5)=    0.00000  celldm(6)=    0.00000

     crystal axes: (cart. coord. in units of a_0)
               a(1) = (  1.0000  0.0000  0.0000 )  
               a(2) = ( -0.5000  0.8660  0.0000 )  
               a(3) = (  0.0000  0.0000  1.0955 )  

     reciprocal axes: (cart. coord. in units 2 pi/a_0)
               b(1) = (  1.0000  0.5774  0.0000 )  
               b(2) = (  0.0000  1.1547  0.0000 )  
               b(3) = (  0.0000  0.0000  0.9128 )  


     Atoms inside the unit cell: 

   Cartesian axes

     site n.  atom      mass           positions (a_0 units)
        1        Mg  24.3050   tau( 1) = (    0.00000    0.00000    0.00000  )
        2        B   11.0000   tau( 2) = (    0.50000    0.28868    0.54777  )
        3        B   11.0000   tau( 3) = (    0.00000    0.57735    0.54777  )

     Computing dynamical matrix for 
                    q = (   0.0000000   0.0000000   0.0000000 )
 
     25 Sym.Ops. (with q -> -q+G )


     G cutoff =   58.3610  (   1835 G-vectors)     FFT grid: ( 16, 16, 18)
     number of k points=  133  gaussian broad. (Ry)=  0.0250     ngauss =   1
                       cart. coord. in units 2pi/a_0
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   0.0011574
        k(    2) = (   0.0000000   0.0000000   0.0760654), wk =   0.0023148
        k(    3) = (   0.0000000   0.0000000   0.1521307), wk =   0.0023148
        k(    4) = (   0.0000000   0.0000000   0.2281961), wk =   0.0023148
        k(    5) = (   0.0000000   0.0000000   0.3042615), wk =   0.0023148
        k(    6) = (   0.0000000   0.0000000   0.3803268), wk =   0.0023148
        k(    7) = (   0.0000000   0.0000000  -0.4563922), wk =   0.0011574
        k(    8) = (   0.0000000   0.0962250   0.0000000), wk =   0.0069444
        k(    9) = (   0.0000000   0.0962250   0.0760654), wk =   0.0138889
        k(   10) = (   0.0000000   0.0962250   0.1521307), wk =   0.0138889
        k(   11) = (   0.0000000   0.0962250   0.2281961), wk =   0.0138889
        k(   12) = (   0.0000000   0.0962250   0.3042615), wk =   0.0138889
        k(   13) = (   0.0000000   0.0962250   0.3803268), wk =   0.0138889
        k(   14) = (   0.0000000   0.0962250  -0.4563922), wk =   0.0069444
        k(   15) = (   0.0000000   0.1924501   0.0000000), wk =   0.0069444
        k(   16) = (   0.0000000   0.1924501   0.0760654), wk =   0.0138889
        k(   17) = (   0.0000000   0.1924501   0.1521307), wk =   0.0138889
        k(   18) = (   0.0000000   0.1924501   0.2281961), wk =   0.0138889
        k(   19) = (   0.0000000   0.1924501   0.3042615), wk =   0.0138889
        k(   20) = (   0.0000000   0.1924501   0.3803268), wk =   0.0138889
        k(   21) = (   0.0000000   0.1924501  -0.4563922), wk =   0.0069444
        k(   22) = (   0.0000000   0.2886751   0.0000000), wk =   0.0069444
        k(   23) = (   0.0000000   0.2886751   0.0760654), wk =   0.0138889
        k(   24) = (   0.0000000   0.2886751   0.1521307), wk =   0.0138889
        k(   25) = (   0.0000000   0.2886751   0.2281961), wk =   0.0138889
        k(   26) = (   0.0000000   0.2886751   0.3042615), wk =   0.0138889
        k(   27) = (   0.0000000   0.2886751   0.3803268), wk =   0.0138889
        k(   28) = (   0.0000000   0.2886751  -0.4563922), wk =   0.0069444
        k(   29) = (   0.0000000   0.3849002   0.0000000), wk =   0.0069444
        k(   30) = (   0.0000000   0.3849002   0.0760654), wk =   0.0138889
        k(   31) = (   0.0000000   0.3849002   0.1521307), wk =   0.0138889
        k(   32) = (   0.0000000   0.3849002   0.2281961), wk =   0.0138889
        k(   33) = (   0.0000000   0.3849002   0.3042615), wk =   0.0138889
        k(   34) = (   0.0000000   0.3849002   0.3803268), wk =   0.0138889
        k(   35) = (   0.0000000   0.3849002  -0.4563922), wk =   0.0069444
        k(   36) = (   0.0000000   0.4811252   0.0000000), wk =   0.0069444
        k(   37) = (   0.0000000   0.4811252   0.0760654), wk =   0.0138889
        k(   38) = (   0.0000000   0.4811252   0.1521307), wk =   0.0138889
        k(   39) = (   0.0000000   0.4811252   0.2281961), wk =   0.0138889
        k(   40) = (   0.0000000   0.4811252   0.3042615), wk =   0.0138889
        k(   41) = (   0.0000000   0.4811252   0.3803268), wk =   0.0138889
        k(   42) = (   0.0000000   0.4811252  -0.4563922), wk =   0.0069444
        k(   43) = (   0.0000000  -0.5773503   0.0000000), wk =   0.0034722
        k(   44) = (   0.0000000  -0.5773503   0.0760654), wk =   0.0069444
        k(   45) = (   0.0000000  -0.5773503   0.1521307), wk =   0.0069444
        k(   46) = (   0.0000000  -0.5773503   0.2281961), wk =   0.0069444
        k(   47) = (   0.0000000  -0.5773503   0.3042615), wk =   0.0069444
        k(   48) = (   0.0000000  -0.5773503   0.3803268), wk =   0.0069444
        k(   49) = (   0.0000000  -0.5773503  -0.4563922), wk =   0.0034722
        k(   50) = (   0.0833333   0.1443376   0.0000000), wk =   0.0069444
        k(   51) = (   0.0833333   0.1443376   0.0760654), wk =   0.0138889
        k(   52) = (   0.0833333   0.1443376   0.1521307), wk =   0.0138889
        k(   53) = (   0.0833333   0.1443376   0.2281961), wk =   0.0138889
        k(   54) = (   0.0833333   0.1443376   0.3042615), wk =   0.0138889
        k(   55) = (   0.0833333   0.1443376   0.3803268), wk =   0.0138889
        k(   56) = (   0.0833333   0.1443376  -0.4563922), wk =   0.0069444
        k(   57) = (   0.0833333   0.2405626   0.0000000), wk =   0.0138889
        k(   58) = (   0.0833333   0.2405626   0.0760654), wk =   0.0277778
        k(   59) = (   0.0833333   0.2405626   0.1521307), wk =   0.0277778
        k(   60) = (   0.0833333   0.2405626   0.2281961), wk =   0.0277778
        k(   61) = (   0.0833333   0.2405626   0.3042615), wk =   0.0277778
        k(   62) = (   0.0833333   0.2405626   0.3803268), wk =   0.0277778
        k(   63) = (   0.0833333   0.2405626  -0.4563922), wk =   0.0138889
        k(   64) = (   0.0833333   0.3367877   0.0000000), wk =   0.0138889
        k(   65) = (   0.0833333   0.3367877   0.0760654), wk =   0.0277778
        k(   66) = (   0.0833333   0.3367877   0.1521307), wk =   0.0277778
        k(   67) = (   0.0833333   0.3367877   0.2281961), wk =   0.0277778
        k(   68) = (   0.0833333   0.3367877   0.3042615), wk =   0.0277778
        k(   69) = (   0.0833333   0.3367877   0.3803268), wk =   0.0277778
        k(   70) = (   0.0833333   0.3367877  -0.4563922), wk =   0.0138889
        k(   71) = (   0.0833333   0.4330127   0.0000000), wk =   0.0138889
        k(   72) = (   0.0833333   0.4330127   0.0760654), wk =   0.0277778
        k(   73) = (   0.0833333   0.4330127   0.1521307), wk =   0.0277778
        k(   74) = (   0.0833333   0.4330127   0.2281961), wk =   0.0277778
        k(   75) = (   0.0833333   0.4330127   0.3042615), wk =   0.0277778
        k(   76) = (   0.0833333   0.4330127   0.3803268), wk =   0.0277778
        k(   77) = (   0.0833333   0.4330127  -0.4563922), wk =   0.0138889
        k(   78) = (   0.0833333   0.5292377   0.0000000), wk =   0.0138889
        k(   79) = (   0.0833333   0.5292377   0.0760654), wk =   0.0277778
        k(   80) = (   0.0833333   0.5292377   0.1521307), wk =   0.0277778
        k(   81) = (   0.0833333   0.5292377   0.2281961), wk =   0.0277778
        k(   82) = (   0.0833333   0.5292377   0.3042615), wk =   0.0277778
        k(   83) = (   0.0833333   0.5292377   0.3803268), wk =   0.0277778
        k(   84) = (   0.0833333   0.5292377  -0.4563922), wk =   0.0138889
        k(   85) = (   0.1666667   0.2886751   0.0000000), wk =   0.0069444
        k(   86) = (   0.1666667   0.2886751   0.0760654), wk =   0.0138889
        k(   87) = (   0.1666667   0.2886751   0.1521307), wk =   0.0138889
        k(   88) = (   0.1666667   0.2886751   0.2281961), wk =   0.0138889
        k(   89) = (   0.1666667   0.2886751   0.3042615), wk =   0.0138889
        k(   90) = (   0.1666667   0.2886751   0.3803268), wk =   0.0138889
        k(   91) = (   0.1666667   0.2886751  -0.4563922), wk =   0.0069444
        k(   92) = (   0.1666667   0.3849002   0.0000000), wk =   0.0138889
        k(   93) = (   0.1666667   0.3849002   0.0760654), wk =   0.0277778
        k(   94) = (   0.1666667   0.3849002   0.1521307), wk =   0.0277778
        k(   95) = (   0.1666667   0.3849002   0.2281961), wk =   0.0277778
        k(   96) = (   0.1666667   0.3849002   0.3042615), wk =   0.0277778
        k(   97) = (   0.1666667   0.3849002   0.3803268), wk =   0.0277778
        k(   98) = (   0.1666667   0.3849002  -0.4563922), wk =   0.0138889
        k(   99) = (   0.1666667   0.4811252   0.0000000), wk =   0.0138889
        k(  100) = (   0.1666667   0.4811252   0.0760654), wk =   0.0277778
        k(  101) = (   0.1666667   0.4811252   0.1521307), wk =   0.0277778
        k(  102) = (   0.1666667   0.4811252   0.2281961), wk =   0.0277778
        k(  103) = (   0.1666667   0.4811252   0.3042615), wk =   0.0277778
        k(  104) = (   0.1666667   0.4811252   0.3803268), wk =   0.0277778
        k(  105) = (   0.1666667   0.4811252  -0.4563922), wk =   0.0138889
        k(  106) = (   0.1666667   0.5773503   0.0000000), wk =   0.0069444
        k(  107) = (   0.1666667   0.5773503   0.0760654), wk =   0.0138889
        k(  108) = (   0.1666667   0.5773503   0.1521307), wk =   0.0138889
        k(  109) = (   0.1666667   0.5773503   0.2281961), wk =   0.0138889
        k(  110) = (   0.1666667   0.5773503   0.3042615), wk =   0.0138889
        k(  111) = (   0.1666667   0.5773503   0.3803268), wk =   0.0138889
        k(  112) = (   0.1666667   0.5773503  -0.4563922), wk =   0.0069444
        k(  113) = (   0.2500000   0.4330127   0.0000000), wk =   0.0069444
        k(  114) = (   0.2500000   0.4330127   0.0760654), wk =   0.0138889
        k(  115) = (   0.2500000   0.4330127   0.1521307), wk =   0.0138889
        k(  116) = (   0.2500000   0.4330127   0.2281961), wk =   0.0138889
        k(  117) = (   0.2500000   0.4330127   0.3042615), wk =   0.0138889
        k(  118) = (   0.2500000   0.4330127   0.3803268), wk =   0.0138889
        k(  119) = (   0.2500000   0.4330127  -0.4563922), wk =   0.0069444
        k(  120) = (   0.2500000   0.5292377   0.0000000), wk =   0.0138889
        k(  121) = (   0.2500000   0.5292377   0.0760654), wk =   0.0277778
        k(  122) = (   0.2500000   0.5292377   0.1521307), wk =   0.0277778
        k(  123) = (   0.2500000   0.5292377   0.2281961), wk =   0.0277778
        k(  124) = (   0.2500000   0.5292377   0.3042615), wk =   0.0277778
        k(  125) = (   0.2500000   0.5292377   0.3803268), wk =   0.0277778
        k(  126) = (   0.2500000   0.5292377  -0.4563922), wk =   0.0138889
        k(  127) = (   0.3333333   0.5773503   0.0000000), wk =   0.0023148
        k(  128) = (   0.3333333   0.5773503   0.0760654), wk =   0.0046296
        k(  129) = (   0.3333333   0.5773503   0.1521307), wk =   0.0046296
        k(  130) = (   0.3333333   0.5773503   0.2281961), wk =   0.0046296
        k(  131) = (   0.3333333   0.5773503   0.3042615), wk =   0.0046296
        k(  132) = (   0.3333333   0.5773503   0.3803268), wk =   0.0046296
        k(  133) = (   0.3333333   0.5773503  -0.4563922), wk =   0.0023148

     PseudoPot. # 1 for Mg read from file mg_6.ncpp
     Pseudo is Norm-conserving + core correction, Zval =  2.0
     Generated by old ld1 code (numerical format)
     Using radial grid of 1140 points,  2 beta functions with: 
                l(1) =   1
                l(2) =   2

     PseudoPot. # 2 for B  read from file B.pbe-n-van_ak.UPF
     Pseudo is Ultrasoft + core correction, Zval =  3.0
     Generated by new atomic code, or converted to UPF format
     Using radial grid of  781 points,  4 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1
     Q(r) pseudized with  8 coefficients,  rinner =    1.100   1.100   1.100



     Atomic displacements:
     There are   6 irreducible representations

     Representation     1      1 modes -B_2g To be done

     Representation     2      2 modes -E_1u To be done

     Representation     3      1 modes -A_2u To be done

     Representation     4      2 modes -E_2g To be done

     Representation     5      2 modes -E_1u To be done

     Representation     6      1 modes -A_2u To be done
     PHONON       :     1.96s CPU time,    2.11s wall time


     Alpha used in Ewald sum =   0.7000


     Representation #  1 mode #   1

     Self-consistent Calculation
 
     Pert. #  1: Fermi energy shift (Ry) =     0.0000E+00     0.0000E+00

      iter #   1 total cpu time :     2.3 secs   av.it.:   3.2
      thresh= 0.100E-01 alpha_mix =  0.700 |ddv_scf|^2 =  0.887E-06
 
     Pert. #  1: Fermi energy shift (Ry) =     0.0000E+00     0.0000E+00

      iter #   2 total cpu time :     2.5 secs   av.it.:   5.9
      thresh= 0.942E-04 alpha_mix =  0.700 |ddv_scf|^2 =  0.715E-07
 
     Pert. #  1: Fermi energy shift (Ry) =     0.0000E+00     0.0000E+00

      iter #   3 total cpu time :     2.6 secs   av.it.:   5.4
      thresh= 0.267E-04 alpha_mix =  0.700 |ddv_scf|^2 =  0.389E-09
 
     Pert. #  1: Fermi energy shift (Ry) =     0.0000E+00     0.0000E+00

      iter #   4 total cpu time :     2.8 secs   av.it.:   6.1
      thresh= 0.197E-05 alpha_mix =  0.700 |ddv_scf|^2 =  0.837E-12

     End of self-consistent calculation

     Convergence has been achieved 


     Representation #  2 modes #   2  3

     Self-consistent Calculation
 
     Pert. #  1: Fermi energy shift (Ry) =     0.0000E+00     0.0000E+00
     Pert. #  2: Fermi energy shift (Ry) =     0.0000E+00     0.0000E+00

      iter #   1 total cpu time :     3.1 secs   av.it.:   3.3
      thresh= 0.100E-01 alpha_mix =  0.700 |ddv_scf|^2 =  0.492E-06
 
     Pert. #  1: Fermi energy shift (Ry) =     0.0000E+00     0.0000E+00
     Pert. #  2: Fermi energy shift (Ry) =     0.0000E+00     0.0000E+00

      iter #   2 total cpu time :     3.4 secs   av.it.:   6.1
      thresh= 0.701E-04 alpha_mix =  0.700 |ddv_scf|^2 =  0.401E-07
 
     Pert. #  1: Fermi energy shift (Ry) =    -0.8616E-26     0.0000E+00
     Pert. #  2: Fermi energy shift (Ry) =     0.8616E-26     0.0000E+00

      iter #   3 total cpu time :     3.8 secs   av.it.:   6.3
      thresh= 0.200E-04 alpha_mix =  0.700 |ddv_scf|^2 =  0.371E-10

     End of self-consistent calculation

     Convergence has been achieved 


     Representation #  3 mode #   4

     Self-consistent Calculation
 
     Pert. #  1: Fermi energy shift (Ry) =     0.0000E+00     0.0000E+00

      iter #   1 total cpu time :     3.9 secs   av.it.:   3.6
      thresh= 0.100E-01 alpha_mix =  0.700 |ddv_scf|^2 =  0.251E-05
 
     Pert. #  1: Fermi energy shift (Ry) =     0.6893E-25     0.0000E+00

      iter #   2 total cpu time :     4.1 secs   av.it.:   6.1
      thresh= 0.158E-03 alpha_mix =  0.700 |ddv_scf|^2 =  0.323E-06
 
     Pert. #  1: Fermi energy shift (Ry) =     0.0000E+00     0.0000E+00

      iter #   3 total cpu time :     4.3 secs   av.it.:   6.1
      thresh= 0.568E-04 alpha_mix =  0.700 |ddv_scf|^2 =  0.261E-07
 
     Pert. #  1: Fermi energy shift (Ry) =     0.0000E+00     0.0000E+00

      iter #   4 total cpu time :     4.4 secs   av.it.:   6.2
      thresh= 0.162E-04 alpha_mix =  0.700 |ddv_scf|^2 =  0.371E-10

     End of self-consistent calculation

     Convergence has been achieved 


     Representation #  4 modes #   5  6

     Self-consistent Calculation
 
     Pert. #  1: Fermi energy shift (Ry) =     0.1156E-17    -0.3737E-44
     Pert. #  2: Fermi energy shift (Ry) =    -0.1446E-18    -0.9342E-44

      iter #   1 total cpu time :     4.8 secs   av.it.:   4.5
      thresh= 0.100E-01 alpha_mix =  0.700 |ddv_scf|^2 =  0.203E-05
 
     Pert. #  1: Fermi energy shift (Ry) =     0.5782E-18     0.1121E-43
     Pert. #  2: Fermi energy shift (Ry) =     0.3180E-17     0.1868E-43

      iter #   2 total cpu time :     5.1 secs   av.it.:   7.4
      thresh= 0.142E-03 alpha_mix =  0.700 |ddv_scf|^2 =  0.533E-07
 
     Pert. #  1: Fermi energy shift (Ry) =     0.5782E-18     0.2392E-42
     Pert. #  2: Fermi energy shift (Ry) =     0.1590E-17     0.1315E-41

      iter #   3 total cpu time :     5.5 secs   av.it.:   7.3
      thresh= 0.231E-04 alpha_mix =  0.700 |ddv_scf|^2 =  0.224E-08
 
     Pert. #  1: Fermi energy shift (Ry) =     0.4337E-18    -0.1196E-42
     Pert. #  2: Fermi energy shift (Ry) =    -0.2891E-18    -0.2093E-42

      iter #   4 total cpu time :     5.8 secs   av.it.:   6.5
      thresh= 0.474E-05 alpha_mix =  0.700 |ddv_scf|^2 =  0.578E-12

     End of self-consistent calculation

     Convergence has been achieved 


     Representation #  5 modes #   7  8

     Self-consistent Calculation
 
     Pert. #  1: Fermi energy shift (Ry) =     0.2262E-25    -0.1254E-36
     Pert. #  2: Fermi energy shift (Ry) =    -0.1239E-25     0.0000E+00

      iter #   1 total cpu time :     6.2 secs   av.it.:   4.1
      thresh= 0.100E-01 alpha_mix =  0.700 |ddv_scf|^2 =  0.449E-06
 
     Pert. #  1: Fermi energy shift (Ry) =     0.5602E-24     0.3429E-38
     Pert. #  2: Fermi energy shift (Ry) =     0.1678E-24     0.0000E+00

      iter #   2 total cpu time :     6.6 secs   av.it.:   7.1
      thresh= 0.670E-04 alpha_mix =  0.700 |ddv_scf|^2 =  0.167E-07
 
     Pert. #  1: Fermi energy shift (Ry) =     0.3447E-25    -0.4898E-39
     Pert. #  2: Fermi energy shift (Ry) =     0.0000E+00     0.0000E+00

      iter #   3 total cpu time :     6.9 secs   av.it.:   6.3
      thresh= 0.129E-04 alpha_mix =  0.700 |ddv_scf|^2 =  0.805E-09
 
     Pert. #  1: Fermi energy shift (Ry) =    -0.4039E-26    -0.9796E-39
     Pert. #  2: Fermi energy shift (Ry) =    -0.2019E-26     0.0000E+00

      iter #   4 total cpu time :     7.2 secs   av.it.:   6.5
      thresh= 0.284E-05 alpha_mix =  0.700 |ddv_scf|^2 =  0.569E-12

     End of self-consistent calculation

     Convergence has been achieved 


     Representation #  6 mode #   9

     Self-consistent Calculation
 
     Pert. #  1: Fermi energy shift (Ry) =     0.0000E+00     0.0000E+00

      iter #   1 total cpu time :     7.4 secs   av.it.:   4.4
      thresh= 0.100E-01 alpha_mix =  0.700 |ddv_scf|^2 =  0.276E-04
 
     Pert. #  1: Fermi energy shift (Ry) =     0.8616E-26     0.3135E-37

      iter #   2 total cpu time :     7.6 secs   av.it.:   6.3
      thresh= 0.525E-03 alpha_mix =  0.700 |ddv_scf|^2 =  0.389E-05
 
     Pert. #  1: Fermi energy shift (Ry) =     0.5515E-24     0.0000E+00

      iter #   3 total cpu time :     7.8 secs   av.it.:   6.1
      thresh= 0.197E-03 alpha_mix =  0.700 |ddv_scf|^2 =  0.178E-07
 
     Pert. #  1: Fermi energy shift (Ry) =     0.6893E-25     0.0000E+00

      iter #   4 total cpu time :     7.9 secs   av.it.:   5.5
      thresh= 0.134E-04 alpha_mix =  0.700 |ddv_scf|^2 =  0.658E-11

     End of self-consistent calculation

     Convergence has been achieved 
 
     Number of q in the star =    1
     List of q in the star:
          1   0.000000000   0.000000000   0.000000000

     Diagonalizing the dynamical matrix

     q = (    0.000000000   0.000000000   0.000000000 ) 

 **************************************************************************
     omega( 1) =      -4.805143 [THz] =    -160.283381 [cm-1]
     omega( 2) =      -2.820097 [THz] =     -94.068941 [cm-1]
     omega( 3) =      -2.820097 [THz] =     -94.068941 [cm-1]
     omega( 4) =       9.001379 [THz] =     300.255680 [cm-1]
     omega( 5) =       9.001379 [THz] =     300.255680 [cm-1]
     omega( 6) =      10.043488 [THz] =     335.016921 [cm-1]
     omega( 7) =      14.484965 [THz] =     483.169631 [cm-1]
     omega( 8) =      14.484965 [THz] =     483.169631 [cm-1]
     omega( 9) =      18.947636 [THz] =     632.029295 [cm-1]
 **************************************************************************

     Mode symmetry, D_6h(6/mmm) point group:

     omega(  1 -  1) =       -160.3  [cm-1]   --> A_2u            I  
     omega(  2 -  3) =        -94.1  [cm-1]   --> E_1u            I  
     omega(  4 -  5) =        300.3  [cm-1]   --> E_1u            I  
     omega(  6 -  6) =        335.0  [cm-1]   --> A_2u            I  
     omega(  7 -  8) =        483.2  [cm-1]   --> E_2g            R  
     omega(  9 -  9) =        632.0  [cm-1]   --> B_2g               

 **************************************************************************
 
     PWSCF        :     6.34s CPU

     Called by init_run:

     Called by electrons:
     v_of_rho     :     0.00s CPU
     newd         :     0.00s CPU

     Called by c_bands:
     init_us_2    :     0.07s CPU (     646 calls,   0.000 s avg)

     Called by *egterg:
     s_psi        :     0.24s CPU (    9402 calls,   0.000 s avg)

     Called by h_psi:
     add_vuspsi   :     0.11s CPU (    4412 calls,   0.000 s avg)

     General routines
     calbec       :     0.50s CPU (   11017 calls,   0.000 s avg)
     cft3s        :     2.86s CPU (   39840 calls,   0.000 s avg)
     davcio       :     0.01s CPU (    2871 calls,   0.000 s avg)
 
     Parallel routines
 
     PHONON       :     7.93s CPU time,    8.50s wall time

     INITIALIZATION: 
     phq_setup    :     0.02s CPU
     phq_init     :     0.34s CPU
 
     phq_init     :     0.34s CPU
     set_drhoc    :     0.26s CPU (       3 calls,   0.088 s avg)
     init_vloc    :     0.00s CPU
     init_us_1    :     0.13s CPU
     newd         :     0.00s CPU
     dvanqq       :     0.02s CPU
     drho         :     0.19s CPU
 
     DYNAMICAL MATRIX:
     dynmat0      :     0.20s CPU
     phqscf       :     5.77s CPU
     dynmatrix    :     0.00s CPU
 
     phqscf       :     5.77s CPU
     solve_linter :     5.65s CPU (       6 calls,   0.941 s avg)
     drhodv       :     0.10s CPU (       6 calls,   0.016 s avg)
 
     dynmat0      :     0.20s CPU
     dynmat_us    :     0.02s CPU
     d2ionq       :     0.00s CPU
     dynmatcc     :     0.18s CPU
 
     dynmat_us    :     0.02s CPU
     addusdynmat  :     0.00s CPU
 
     phqscf       :     5.77s CPU
     solve_linter :     5.65s CPU (       6 calls,   0.941 s avg)
 
     solve_linter :     5.65s CPU (       6 calls,   0.941 s avg)
     dvqpsi_us    :     0.43s CPU (     153 calls,   0.003 s avg)
     ortho        :     0.15s CPU (     578 calls,   0.000 s avg)
     cgsolve      :     2.89s CPU (     578 calls,   0.005 s avg)
     incdrhoscf   :     0.41s CPU (     578 calls,   0.001 s avg)
     addusddens   :     0.08s CPU (      29 calls,   0.003 s avg)
     vpsifft      :     0.30s CPU (     425 calls,   0.001 s avg)
     dv_of_drho   :     0.08s CPU (      34 calls,   0.002 s avg)
     mix_pot      :     0.01s CPU (      23 calls,   0.000 s avg)
     ef_shift     :     0.01s CPU (      29 calls,   0.000 s avg)
     localdos     :     0.09s CPU (       6 calls,   0.015 s avg)
     psymdvscf    :     0.12s CPU (      23 calls,   0.005 s avg)
     newdq        :     0.08s CPU (      23 calls,   0.003 s avg)
     adddvscf     :     0.03s CPU (     425 calls,   0.000 s avg)
     drhodvus     :     0.00s CPU (       6 calls,   0.000 s avg)
 
     dvqpsi_us    :     0.43s CPU (     153 calls,   0.003 s avg)
     dvqpsi_us_on :     0.08s CPU (     153 calls,   0.001 s avg)
 
     cgsolve      :     2.89s CPU (     578 calls,   0.005 s avg)
     ch_psi       :     2.83s CPU (    4412 calls,   0.001 s avg)
 
     ch_psi       :     2.83s CPU (    4412 calls,   0.001 s avg)
     h_psiq       :     2.42s CPU (    4412 calls,   0.001 s avg)
     last         :     0.38s CPU (    4412 calls,   0.000 s avg)
 
     h_psiq       :     2.42s CPU (    4412 calls,   0.001 s avg)
     firstfft     :     0.89s CPU (   12979 calls,   0.000 s avg)
     secondfft    :     1.01s CPU (   12979 calls,   0.000 s avg)
     add_vuspsi   :     0.11s CPU (    4412 calls,   0.000 s avg)
 
     incdrhoscf   :     0.41s CPU (     578 calls,   0.001 s avg)
     addusdbec    :     0.05s CPU (     731 calls,   0.000 s avg)
 
     drhodvus     :     0.00s CPU (       6 calls,   0.000 s avg)
 
      General routines
     calbec       :     0.50s CPU (   11017 calls,   0.000 s avg)
     cft3s        :     2.86s CPU (   39840 calls,   0.000 s avg)
     davcio       :     0.01s CPU (    2871 calls,   0.000 s avg)
     write_rec    :     0.12s CPU (      29 calls,   0.004 s avg)
 
